packages/edit-post/src/components/header/template-title/index.js

```diff
@@ -36,32 +36,37 @@ function TemplateTitle() {
 	}
 
 	return (
-		<Dropdown
-			position="bottom center"
-			className="edit-post-template-top-area"
-			contentClassName="edit-post-template-top-area__popover"
-			renderToggle={ ( { onToggle } ) => (
-				<>
-					<div className="edit-post-template-title">
-						{ __( 'About' ) }
-					</div>
-					<Button
-						isSmall
-						isTertiary
-						onClick={ onToggle }
-						aria-label={ __( 'Template Options' ) }
-					>
-						{ templateTitle }
-					</Button>
-				</>
-			) }
-			renderContent={ () => (
-				<>
-					<EditTemplateTitle />
-					<DeleteTemplate />
-				</>
-			) }
-		/>
+		<ToolbarItem>
+			{ ( toolbarItemHTMLProps ) => {
+				return (
+					<Dropdown
+						className="edit-post-template-top-area"
+						position="bottom center"
+						contentClassName="edit-post-template-top-area__popover"
+						renderToggle={ ( { onToggle } ) => (
+							<>
+								<div>{ __( 'About' ) }</div>
+								<Button
```

Comment:
I think you could just have used `ToolbarButton` here no? Maybe the warning is too strong?

Reply:
I'm almost sure it was one of the (many) things I tried and the warning persisted.